source/patterns/@aws-solutions-constructs/aws-lambda-elasticsearch-kibana/test/lambda-elasticsearch-kibana.test.ts

```diff
@@ -15,14 +15,11 @@ import { LambdaToElasticSearchAndKibana, LambdaToElasticSearchAndKibanaProps } f
 import * as lambda from '@aws-cdk/aws-lambda';
 import * as cdk from "@aws-cdk/core";
 import '@aws-cdk/assert/jest';
+import * as defaults from '@aws-solutions-constructs/core';
 
 function deployNewFunc(stack: cdk.Stack) {
   const props: LambdaToElasticSearchAndKibanaProps = {
-    lambdaFunctionProps: {
-      code: lambda.Code.fromAsset(`${__dirname}/lambda`),
-      runtime: lambda.Runtime.NODEJS_14_X,
-      handler: 'index.handler'
-    },
+    lambdaFunctionProps: getDefaultLambdaProps(),
```

Comment:
Let's call this getDefaultTestLambdaProps() to avoid confusion with lambda-defaults.ts